cmis/src/test/java/nl/nn/adapterframework/extensions/cmis/CmisTestObject.java

```diff
@@ -52,10 +53,10 @@ public CmisObject answer(InvocationOnMock invocation) throws Throwable {
 
 	@Override
 	public String getId() {
-		return objectId;
+		return Base64.encodeBase64String(objectId.getBytes());
 	}
 	public String getObjectId() {
-		return objectId;
+		return Base64.encodeBase64String(objectId.getBytes());
```

Comment:
Ik kan me voorstellen dat het zinvol is om een verschil te maken tussen id en objectId

Reply:
Volgens de spec:
[String](http://docs.oracle.com/javase/7/docs/api/java/lang/String.html?is-external=true) getId() -> Returns the object ID.

Reply:
hmm

Reply:
Zie https://chemistry.apache.org/java/0.11.0/maven/apidocs/org/apache/chemistry/opencmis/client/api/ObjectId.html#getId()

Reply:
Maar als ze dan toch hetzelfde zijn, waarom is dan getObjectId() nog nodig?